ui/components/multichain/pages/send/send.js

```diff
@@ -405,14 +406,16 @@ export const SendPage = () => {
             onClick={onCancel}
           />
         }
+        endAccessory={
+          remoteModeSupported && (
+            <SendAllowanceTooltip allowance={remoteSendAllowance} />
```

Comment:
As `remoteSendAllowance` isn't used anywhere else, could we encapsulate this and the selector usage inside `SendAllowanceTooltip`?